ppa/archive/import_util.py

```diff
@@ -247,7 +255,15 @@ def pairtree_paths(self):
     def rsync_data(self):
         """Use rsync to retrieve data for the volumes to be imported."""
 
-        logger.info("rsyncing pairtree data for %s", ", ".join(self.source_ids))
+        # limit the number of ids included in the log message
+        log_detail = ""
+        rsync_count = len(self.source_ids)
+        if rsync_count <= 12:
```

Comment:
Suggestion: switch to 10 since it's typically interpreted as less specific than 12.
```suggestion
        if rsync_count <= 10:
```